packages/smart-contracts/src/contracts/ERC20EscrowToPay.sol

```diff
@@ -26,67 +27,79 @@ contract ERC20EscrowToPay {
     }
 
     /**
-    * @notice Mapping is used to store the Requests in escrow. 
-    */
+     * @notice Mapping is used to store the Requests in escrow.
+     */
     mapping(bytes => Request) public requestMapping;
 
     /**
-    * @notice Modifier checks if msg.sender is the requestpayment payer.
-    * @param _paymentRef Reference of the requestpayment related.
-    * @dev It requires msg.sender to be equal to requestMapping[_paymentRef].payer. 
-    */
+     * @notice Duration of emergency claim period that payee can initiate.
+     The payer can reverse this claim within this period.
```

Comment:
To align with format from
https://github.com/RequestNetwork/requestNetwork/blob/eb28a76064dd874c812ec97dd5dbb53a845af5b2/packages/smart-contracts/src/contracts/EthConversionProxy.sol#L8-L14

```suggestion
     * @notice Duration of emergency claim period that payee can initiate.
     *         The payer can reverse this claim within this period.
```